### PR TITLE
Added function to save the canvas in its present state.

### PR DESCRIPTION
**Pull request by sam-peach**
The canvas can now take drawings from multiple clients, and once 'p' is pressed this will save the canvas as a jpeg to that client.

Very clunky for right now, but I'm guessing this would be what would need to be saved to the database...?

### CODE REVIEW

File: public/sketch.js [hunk 1]
```diff
@@ -2,13 +2,14 @@ const socket = io.connect('http://localhost:3000');
 
 function setup(){
 	cursor(CROSS);
+	pixelDensity(1); //Ensures that all displays show the same pixel density.
```
Comment (jffhtchr) [inline public/sketch.js:5]: nice!

File: public/sketch.js [hunk 2]
```diff
@@ -34,3 +35,29 @@ function mouseDataSend() {
 		socket.emit('mouse', data);
 	}
 }
+
+// If you press 'p' on your keyboard this function will save the canvas
+// in its current state to a jpg in your downloads.
+// This is what we'd want to send to the database.
+function keyPressed(event) {
```
Comment (jffhtchr) [inline public/sketch.js:42]: this is crazy! all native p5.js stuff??

**Review by jffhtchr — APPROVED** [review]

**Comment by jffhtchr** [comment]
Merge away dooood

File: public/sketch.js (hunk 2 above)
Comment (sam-peach) [reply public/sketch.js:42]: @jffhtchr Yep! More than a few times I've custom made an awkward function only to find that p5 has a built-in function, haha.

**Pull request merged**